### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by alexjurkiewicz**

### CODE REVIEW

File: crawl-ref/INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+  * [.des Level Compiler](#des-level-compiler)
+  * [Code Coverage](#code-coverage)
+  * [Lua](#lua)
+  * [PCRE](#pcre)
+  * [Unicode](#unicode)
+* [Getting Help](#getting-help)
+
+## Getting DCSS To Run
+
+You can download prebuild versions of DCSS from the [DCSS homepage](https://crawl.develz.org/download.htm).
```
Comment (amalloy) [inline crawl-ref/INSTALL.md]: prebuilt

**Comment by Goratrix** [comment]
One tiny change I would suggest for the MSYS2 instructions, MSYS2 has recently (2 weeks ago) fully retired python 2 and renamed all packages, so the 'python3' names are no longer needed, it's all just 'python' now. Also the folder name is not needed there, so lines 213-216 can just say:

>  pacman -S mingw-w64-x86_64-python-yaml
    # or
    pacman -S mingw-w64-x86_64-python-pip
    pip install pyyaml

File: crawl-ref/INSTALL.md (hunk 1 above)
Comment (alexjurkiewicz) [reply crawl-ref/INSTALL.md]: whoops

**Comment by alexjurkiewicz** [comment]
I don't think there's any outstanding feedback (including from @gammafunk in IRC), let me know if you would like further changes.

File: README.md [hunk 2]
```diff
@@ -32,11 +32,13 @@ Both classical ASCII and tiles (GUI) versions of Crawl are available to [downloa
...
 
-### Web
-Our official homepage is [https://crawl.develz.org](https://crawl.develz.org/). You can find online play, offline downloads, a community forum and the development bug tracker/wiki.
-
-### IRC
-Chat with fellow crawlers in [##crawl on irc.freenode.net](https://webchat.freenode.net/?channels=##crawl), or talk development in [##crawl-dev](https://webchat.freenode.net/?channels=##crawl-dev).
+* Our official homepage: [crawl.develz.org](https://crawl.develz.org/)
+  * Online webtiles play
+  * Offline downloads
+  * A community forum.
```
Comment (aidanholm) [inline README.md]: inconsistent full stop here

File: README.md [hunk 3]
```diff
@@ -32,11 +32,13 @@ Both classical ASCII and tiles (GUI) versions of Crawl are available to [downloa
...
+* The official [Reddit /r/dcss subreddit](https://www.reddit.com/r/dcss/)
+* [Reddit /r/roguelikes Discord](https://discord.gg/S5F2H32)
```
Comment (aidanholm) [inline README.md]: probably just [/r/dcss subreddit] is specific enough, and likewise [/r/roguelikes subreddit Discord]

**Comment by ebering** [comment]
Getting the e-mail notifications about this reminded me that I downloaded it, rebased and tweaked it, and then never merged and pushed. I'll do that this evening.

**Pull request closed without merging**